### PR TITLE
Elevator function v0.1 is ready

### PR DESCRIPTION
**Pull request by TatsuyaTsubakimoto**
I've done the stuff. please, review ASAP

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1 +1,2 @@
 console.log('first version');
+console.log('edited by tt');
```
Comment (TatsuyaTsubakimoto) [inline script.js:2]: add another log output

File: script.js [hunk 2]
```diff
@@ -1 +1,2 @@
 console.log('first version');
```
Comment (TatsuyaTsubakimoto) [inline script.js:1]: this is cool

**Review by totmansan — CHANGES_REQUESTED** [review]
check it out

File: script.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (TatsuyaTsubakimoto) [reply script.js:2]: done it

**Review by totmansan — APPROVED** [review]

**Review by totmansan — APPROVED** [review]

**Pull request merged**